plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -791,6 +791,15 @@ var _ = Describe("validator", func() {
 				Expect(err).To(BeForbiddenError())
 				Expect(err).To(MatchError(ContainSubstring("spec.seedName cannot be changed by patching the shoot, please use the shoots/binding subresource")))
 			})
+
+			It("should reject update of binding when shoot.spec.seedName is not nil and the binding has the same seedName", func() {
+				attrs := admission.NewAttributesRecord(&shoot, &oldShoot, core.Kind("Shoot").WithVersion("version"), shoot.Namespace, shoot.Name, core.Resource("shoot").WithVersion("version"), "binding", admission.Update, &metav1.UpdateOptions{}, false, nil)
```

Comment:
I see that you changed `shoot` to `shoots` in other places where `core.Resource("shoot").WithVersion("version")` is called, but not here.